main.js

```diff
@@ -196,31 +196,36 @@ function startCountDown() {
   }
 }
 
+function returnFromLocalStorage() {
```

Comment:
We will have to come back and refactor this

Reply:
agreed!